src/features/user/TreeMapper/Import/components/SampleTrees.tsx

```diff
@@ -67,15 +99,30 @@ export default function SampleTrees({
         Papa.parse(csv, {
           header: true,
           complete: (results) => {
-            //validation for csv file needs to be added
-            const resultsData: SampleTree[] = results.data;
-            const sampleTrees: SampleTree[] = resultsData.map((sampleTree) => {
-              return {
-                ...sampleTree,
-                otherSpecies: sampleTree.otherSpecies,
-              };
-            });
-            addSampleTrees(sampleTrees);
+            const parsedHeaders = results.meta.fields || [];
+            const resultsData = results.data as SampleTree[];
```

Comment:
Instead of casting to specify the type of results data, you can use

`Papa.parse<SampleTree>()` instead of `Papa.parse()` as this specifies the type of result data.